actions/transactions/broadcastCallback.go

```diff
@@ -0,0 +1,42 @@
+package transactions
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/bitcoin-sv/go-broadcast-client/broadcast"
+	"github.com/julienschmidt/httprouter"
+	apirouter "github.com/mrz1836/go-api-router"
+)
+
+// broadcastCallback will handle a broadcastCallback call from the broadcast api
+// Broadcast Callback godoc
+// @Summary		Broadcast Callback
+// @Tags		Transactions
+// @Param 		transaction body broadcast.SubmittedTx true "transaction"
+// @Success		200
+// @Router		/transaction/broadcast/callback [post]
+// @Security	callback-auth
+func (a *Action) broadcastCallback(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	var resp *broadcast.SubmittedTx
+
+	err := json.NewDecoder(req.Body).Decode(&resp)
+	if err != nil {
+		apirouter.ReturnResponse(w, req, http.StatusExpectationFailed, err.Error())
+		return
+	}
+
+	defer req.Body.Close()
+
+	err = a.Services.Bux.UpdateTransaction(req.Context(), resp)
+	if err != nil {
+		msg := fmt.Sprintf("failed to update transaction - tx: %v", resp)
+		a.Services.Logger.Err(err).Msg(msg)
+		apirouter.ReturnResponse(w, req, http.StatusOK, "")
```

Comment:
Are you sure to return `StatusOK` in case of the error?

Reply:
Yes, we always want to response with StatusOK to callback as Arc doesn't handle responses for callback - it's "one way ticket". 
But for the clarication I can add 500 or something.

Reply:
fixed